mkdocs.yml

```diff
@@ -8,59 +8,61 @@ repo_url: https://github.com/V1EngineeringInc/V1EngineeringInc-Docs/
 edit_uri: 'tree/master/docs/'
 
 nav:
- - Index: index.md
- - Editing mkdocs: mkdocs_info.md
+ - New Documents Home: index.md
 
- - MPCNC:
-   - Main: mpcnc/index.md
-   - Assembly:
-     - Table: mpcnc/table.md
-     - Conduit: mpcnc/conduit.md
-     - Corners, Rollers, Base: mpcnc/base.md
-     - Z-Axis: mpcnc/z-axis.md
-     - Middle Assembly: mpcnc/middle-assm.md
-     - Gantry: mpcnc/gantry.md
-     - Final: mpcnc/final.md
-     - Belts: mpcnc/belts.md
+ - Build:
+   - MPCNC: 
+     - Basics: mpcnc/index.md
+     - Instructions:
+       - Table: mpcnc/table.md
+       - Conduit: mpcnc/conduit.md
+       - Corners, Rollers, Base: mpcnc/base.md
+       - Z-Axis: mpcnc/z-axis.md
+       - Middle Assembly: mpcnc/middle-assm.md
+       - Gantry: mpcnc/gantry.md
+       - Final: mpcnc/final.md
+       - Belts: mpcnc/belts.md
+     - Trouble Shooting: mpcnc/troubleshooting.md
+   - LowRider:
+     - Assembly: lowrider/index.md
+     - Old Version: lowrider/version1.md
+   - MP3DP:
+     - Main: mp3dp/index.md
+     - Old Version: mp3dp/version1.md
+   - ZenXY: zenxy/index.md
 
- - LowRider:
-   - Assembly: lowrider/index.md
-
- - MP3DP:
-   - Main: mp3dp/index.md
-   - Old Version: mp3dp/version1.md
-
- - Software:
-   - Main: software/index.md
-   - Repetier-Host: software/repetier-host.md
-   - EstlCAM Basics: software/estlcam-basics.md
-   - EstlCAM Intermediate (2.5d): software/estlcam-2p5d.md
-
- - Electronics:
-   - Main: electronics/index.md
-   - Electrical:
+ - Control:
+   - Some Control Board Options:
      - Ultimachine Boards: electronics/ultimachine.md
      - Ramps: electronics/ramps.md
-     - Steppers: electronics/steppers.md
-     - Reverse Stepper: software/reverse-motor.md
-   - Getting Started:
-     - Software: software/index.md
-     - Firmware: electronics/marlin-firmware.md
+   - Steppers:
+     - Wiring The Steppers: electronics/steppers.md
+   - Some Advanced Options:
+     - v1pi - Headless: electronics/v1pi.md

```

Comment:
I would also be OK if we moved v1pi to the Learn or Use section.

Reply:
I kinda agree. There are a few pages that should be split. Now that I think the categories make more sense. Even the Zen page needs to be split.
So this v1pi page should be more what it is and what is possible, in learn it should have details and troubleshooting and stuff. I really want to add to the options, like an optional features page, Jamie's joystick, game pads, grbl, esp32 stuff.

Reply:
So leave it here for now and until we split it or add some sort of placeholder (quick features page)? I think this is a the more traffic spot. I put headless instead of wireless because I thought that would draw more attention but now it looks a little odd.

Reply:
I created an issue we can come back to in the future.